index.js

```diff
@@ -88,10 +88,6 @@ for (const platform of platforms) {
                 platformConfig.features[key].exceptions = platformConfig.features[key].exceptions.concat(platformOverride.features[key].exceptions)
             }
         }
-        if (isFeatureDisabled(platformConfig.features[key])) {
```

Comment:
hm, I thought we commented out this code out because missing feature keys were causing issues?

Reply:
Just needed a rebase, was done at the same time github wasn't helping as it was a clean rebase.